examples/cython_polar/CMakeLists.txt

```diff
@@ -0,0 +1,24 @@
+cmake_minimum_required(VERSION 3.2)
+cmake_policy(SET CMP0054 NEW)
+
+project (codec)
```

Comment:
`cython_polar` or `codec_polar` could be better, no?

Reply:
Ack.

Reply:
Done.